packages/toolkit/src/createAsyncThunk.ts

```diff
@@ -125,15 +118,10 @@ export type AsyncThunkConfig = {
   rejectedMeta?: unknown
 }
 
-type GetState<ThunkApiConfig> = ThunkApiConfig extends {
-  state: infer State
-}
-  ? State
-  : unknown
 type GetExtra<ThunkApiConfig> = ThunkApiConfig extends { extra: infer Extra }
   ? Extra
   : unknown
-type GetDispatch<ThunkApiConfig> = ThunkApiConfig extends {
+type GetDispatchType<ThunkApiConfig> = ThunkApiConfig extends {
```

Comment:
:question: what required the renames?

Reply:
This one was done to avoid name collision with `GetDispatch` type in `src/dynamicMiddleware/types.ts`.

Reply:
Same thing. I'd rather keep the `cAT` version as the "primary" - seems like there'd be less churn overall, and `dynamicMiddleware` is both a newer file and less important.

Reply:
Sure thing

Reply:
The only issue is the `GetDispatch` type from `src/dynamicMiddleware/types.ts` is the one that is exported through the main entry point, so if I rename that one, it'll be a breaking change.